service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -189,8 +196,15 @@ public ResponseEntity<Void> deleteGcsObjectReference(UUID workspaceUuid, UUID re
     AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
     workspaceService.validateWorkspaceAndAction(
         userRequest, workspaceUuid, SamWorkspaceAction.DELETE_REFERENCE);
+    var userStatusInfo =
```

Comment:
In general, we don't use `var` if the type is not obvious. In this case, I do not know the type.

Reply:
the method is called getUserStatusinfo though :) i specified the type